lib/witchcraft/applicative.ex

```diff
@@ -31,9 +31,11 @@ defclass Witchcraft.Applicative do
   @type t :: any()
 
   defmacro __using__(opts \\ []) do
+    module_imports = [except: Keyword.get(opts, :except, [])]
```

Comment:
As usual, these changes look great 👍 

Doesn't this leave out all other import options, though? For example, `:only`?

Reply:
Also, any thoughts on extracting this common functionality inside the `__using__` out into a `.Internal` module? May be premature optimization, but means we don't have to trace down all of these spots in the future.